contracts/NominationDAO.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: GPL-3.0-only
+// This is a PoC to use the staking precompile wrapper as a Solidity developer.
+pragma solidity >=0.8.0;
+
+import "./StakingInterface.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+contract NominationDAO is AccessControl {
+    using SafeMath for uint256;
+    
+    event Log(uint _value,uint _value2);
+
+    // TODO Our interface should have an accessor for this.
+    uint256 public constant MinNominatorStk = 5 ether;
```

Comment:
We have this now https://github.com/PureStake/moonbeam/blob/master/precompiles/parachain-staking/StakingInterface.sol#L26

This expanded nomination DAO is awesome!

Reply:
Nice! I was looking at the old interfae when I wrote this